### PR TITLE
[SW-504, SW-1000] Do not depend on http compiler

### PR DESCRIPTION
**Pull request by CedrikNikita**

### CODE REVIEW

**Comment by Liubov-crypto** [comment]
LGTM

**Review by subhod-i — CHANGES_REQUESTED** [review]
There are translation keywords related to the compiler. 
`networkCompilerPlaceholder`
`networkCompilerLabel`

These could be removed.

**Comment by subhod-i** [comment]
Great PR.

File: src/popup/components/Modals/ConfirmTransactionSign.vue [hunk 1]
```diff
@@ -408,11 +410,21 @@ export default defineComponent({
...
-            { body: { bytecode, calldata: popupProps.value.tx.callData } },
-          );
-          txFunction.value = txParams.function as TxFunctionRaw;
+          const bytecodeContractCallEncoder = new BytecodeContractCallEncoder(bytecode);
+
+          let txParams = bytecodeContractCallEncoder.decodeCall(
+            popupProps.value.tx.callData,
+          ) as any;
+
+          txParams = {
```
Comment (peronczyk) [inline src/popup/components/Modals/ConfirmTransactionSign.vue:419]: I suggest to have different variable name here instead of reassigning it.

**Review by peronczyk — APPROVED** [review]

**Review by subhod-i — APPROVED** [review]

**Comment by Liubov-crypto** [comment]
LGTM

**Pull request merged**